Update Visual Studio extention documentation (#2287)
telerik/blazor-docs: getting-started/vs-integration/convert-project-wizard.md

![Start the Convert Project Wizard](images/vs-ext-convert-project-entry.png)

> ## Converting the Project Thought the Wizard

Reviewer: The article is all about the wizard so you can afford to skip the word "Wizard" in this heading for brevity.
```suggestion
## Converting the Project
```

> Clicking the Convert to Telerik Application button opens the wizard and you will go through the following steps:

Reviewer: Avoid introducing sections in the way you introduce list items.
```suggestion
Clicking the Convert to Telerik Application button opens the Convert Project Wizard and you will go through the following steps:
1. [Configure the project](#step-1-select-initial-project-configuration).
1. [Select a visual theme](#step-2-select-theme).
1. [Create a backup](#step-3-create-backup).
```

### Step 1: Select Initial Project Configuration

> In this screen, you can choose:

Reviewer 2: This sentence does not correspond to the items below. For example, you get "you can choose ... A version selector". Consider something like "This screen provides the ability to:" and then start with verbs:

* Download ...
* Select a version ...
* Configure additional project settings like ....

* The ability to download the latest available version.
* A version selector so you can choose a particular version to add to the project.
* Additional project settings like:

>     * Whether to import the required Telerik UI stylesheet and JS Interop file by using the [static assets]({%slug getting-started/what-you-need%}#css-theme-and-javascript-files) or the [CDN method]({%slug common-features-cdn%}).

Reviewer 2: ```suggestion
    * Whether to import the required Telerik UI stylesheet and JS Interop file as [static assets]({%slug getting-started/what-you-need%}#css-theme-and-javascript-files) or from the [Telerik CDN]({%slug common-features-cdn%}).
```

>     * Whether to use [localization]({%slug globalization-localization%}) in the project or not.

Reviewer 2: ```suggestion
    * Enable [localization]({%slug globalization-localization%}).
```

>     * Whether to use [Font Icons]({%slug common-features-icons%}) in the project or the default SVG icons.

Reviewer 2: ```suggestion
    * Use [font icons]({%slug common-features-icons%}) instead of the default SVG icons.
```

### Step 2: Select Theme

In this step, you can apply one of the available [themes]({%slug themes-built-in%}) to your project.

### Step 3: Create a Backup

> Whether to back up the project state before the conversion (in a folder called `<ProjectName>_Backup>_<timestamp>`).

Reviewer 2: This sounds Bulgarian. Consider something like:

```suggestion
You can choose to back up the project state before the conversion (in a folder called `<ProjectName>_Backup>_<timestamp>`).
```

### Error Screen

> If you see an error screen in the wizard, make sure that the actual Blazor application is selected in the Solution Explorer. This is most common with WebAssembly types of applications as they consist of a server and shared projects by default, and only the client project is the actual Blazor app that needs the Telerik components.

Reviewer 2: ```suggestion
If you see an error screen in the wizard, make sure that you have selected the UI project in the Solution Explorer. Issues are common with WebAssembly apps as they can include a Server and Shared projects, and only the Client project is the actual Blazor UI app that needs the Telerik components.
```
